packages/kbn-optimizer/package.json

```diff
@@ -29,6 +29,7 @@
     "css-loader": "^3.4.2",
     "del": "^5.1.0",
     "execa": "^4.0.2",
+    "fast-glob": "^3.2.4",
```

Comment:
What's the reason to use another package for glob and not use one which is already used in the repo?

Reply:
`fast-glob` is the package that `globby` wraps, and it provides results merged with `Fs.Stats` objects so that we don't have to use `fs.statSync` a second time.

Reply:
Ultimately is shouldn't cause additional installs if I use the same version that globby is using under the hood.

Reply:
[`4201fb6` (#71319)](https://github.com/elastic/kibana/pull/71319/commits/4201fb60b66bf59dd9e50dab9d0ff66131df8974)

Reply:
Alright, I just removed the use of glob all together and recursively called `readdirSync()` and `statSync()`